Packs/CommonScripts/ReleaseNotes/1_19_46.md

```diff
@@ -0,0 +1,10 @@
+
+#### Scripts
+
+##### ConvertFile
+
+- Updated the ConvertFile script to handle invalid files by returning a failure error within the file context instead of causing a script failure.
+
+##### New: CheckPDFEncryptionAndValidity
+
+- New: Added a new script- CheckPDFEncryptionAndValidity that Returns if a PDF is valid and if it is encrypted.
```

Comment:
```suggestion
Updated the ConvertFile script to handle invalid files by returning a failure error within the file context instead of causing a script failure.

##### New: CheckPDFEncryptionAndValidity

New: Added a new script- CheckPDFEncryptionAndValidity that Returns if a PDF is valid and if it is encrypted.
```